docs/modules/maintain-cluster/pages/enterprise-rest-api.adoc

```diff
@@ -0,0 +1,411 @@
+            <certificate-key>/path/to/certificate-key</certificate-key>
+            <trust-certificate>/path/to/trust-certificate</trust-certificate>
+            <trust-certificate-key>/path/to/trust-certificate-key</trust-certificate-key>
+        </ssl>
+    </rest>
+</hazelcast>
+----
+--
+
+YAML::
++
+[source,yaml]
+----
+hazelcast:
+  rest:
+    enabled: true
+    ssl:
+      enabled: true
+      client-auth: NEED
+      ciphers: TLS_RSA_WITH_AES_128_CBC_SHA, TLS_RSA_WITH_AES_128_CBC_SHA256
+      enabled-protocols: TLSv1.2, TLSv1.3
+      key-alias: myKeyAlias
+      key-password: myKeyPassword
+      key-store: /path/to/keystore
+      key-store-password: myKeyStorePassword
+      key-store-type: JKS
+      key-store-provider: SUN
+      trust-store: /path/to/truststore
+      trust-store-password: myTrustStorePassword
+      trust-store-type: JKS
+      trust-store-provider: SUN
+      protocol: TLS
+      certificate: /path/to/certificate
+      certificate-key: /path/to/certificate-key
+      trust-certificate: /path/to/trust-certificate
+      trust-certificate-key: /path/to/trust-certificate-key
+----
+====
+
+
+== JWT Based Authorization
+
+After a successful authentication by making request to token endpoint, you will receive a short-living JWT token with assigned role names as a claim for authorization checks in subsequent REST calls.
+
+Token issued by one member is not trusted by other members.
+
+Tokens are valid for 15 minutes by default. To change it you can use the rest configuration as shown in the below declarative example:
+
+[tabs]
+====
+XML::
++
+--
+[source,xml]
+----
+<hazelcast>
+    <rest enabled="true">
+        <token-validity-seconds>300</token-validity-seconds>
+    </rest>
+</hazelcast>
+----
+--
+
+YAML::
++
+[source,yaml]
+----
+hazelcast:
+  rest:
+    enabled: true
+    token-validity-seconds: 300
+----
+====
+
+== Obtaining a Token
+
+To obtain a token, you should send a `POST` request to the token endpoint at `/hazelcast/rest/api/v1/token`. The request should include the `username` and `password` in JSON format. If authentication successful, the response will contain a token.
+
+Example request using cURL:
+
+[source,shell]
+----
+curl -X 'POST' \
+  'http://localhost:8443/hazelcast/rest/api/v1/token' \
+  -H 'Content-Type: application/json' \
+  -d '{
+  "username": "restuser",
+  "password": "restpassword"
+}'
+----
+
+
+== GET/POST/DELETE HTTP Request Examples
+
+All the requests in the REST API can return one of the following types of responses.
+
+* Successful Void/Boolean response which does not have body.
+
+* Successful response which returns data in JSON format. For example:
+[source,json]
+----
+{
+  "nodeState": "ACTIVE",
+  "clusterState": "ACTIVE",
+  "numberOfMembers": 1
+}
+----
+
+* Error response in JSON format. For example:
+[source,json]
+----
+{
+  "statusCode": 400,
+  "message": "Please provide a valid value."
+}
+----
+
+
+=== Retrieving Cluster Status
```

Comment:
Can you put a section about how the user can use the swagger pages?
We can describe that all the available APIs are documented at the swagger pages based on their categories together with examples. You can also try the APIs from the swagger ui page.

After this detail swagger description, and may be an example swagger ui snapshot, we may not need each individual endpoint described here but refer to the swagger page for each API, what do you think? This way we do not repeat the same info at the swagger page and no need for duplicate maintenance.

Reply:
fixed at https://github.com/hazelcast/hz-docs/pull/1097/commits/209e01b9b0a9ff8f628a99d596734a645d619882